lib/src/fonts.dart

```diff
@@ -1,101 +1,91 @@
 import "package:flutter/material.dart";
 
+/// Pixels per point constant.
+/// In CSS:
+/// pt = 1/72 inch
+/// px = 1/96 inch
+const double pixelsPerPoint = 96.0 / 72.0; // 1.333(3)
+
 /// The text fonts supported on aRdent smart glasses
 @immutable
 class GYWFont {
-  /// The name of the font
+  /// The name of the font.
   final String name;
 
-  /// The internal prefix of the font
-  final String prefix;
+  /// The font filename on the GYW device (5 characters-long and no type extension).
+  final String filename;
 
-  /// The size (in points) of a character
-  final int size;
+  /// Whether the fontweight is bold (w700) or normal (w400).
+  final bool bold;
 
-  /// The width (in pixels) of a character
-  final int width;
+  /// Whether the font style is italic.
+  final bool italic;
 
-  /// The height (in pixels) of a character
-  final int height;
+  /// The path to the TTF file in the assets folder.
+  final String pathTtf;
 
-  /// Whether the fontweight is bold (w700) or normal (w400)
-  final bool bold;
+  /// The aspect ratio of the font.
+  final double aspectRatio;
 
   /// Creates a font.
   const GYWFont({
     required this.name,
-    required this.prefix,
-    required this.size,
-    required this.width,
-    required this.height,
-    required this.bold,
-  });
+    required this.filename,
+    required this.pathTtf,
+    required this.aspectRatio,
```

Comment:
You can give a default value to these. The goal is to be able to create a `GYWFont` object based only on the filename, in a `TextDrawing.fromJson` method for example.